scripts/pages/pgPeopleLinkDetail.ts

```diff
@@ -0,0 +1,66 @@
+import PgPeopleLinkDetailDesign from 'generated/pages/pgPeopleLinkDetail';
+import LviProperty from 'components/LviProperty';
+import { execute } from 'services/people'
+
+export default class PgPeopleLinkDetail extends PgPeopleLinkDetailDesign {
+    key: string;
```

Comment:
There is a small risk of those variables being undefined. You might want to handle them with `key = ''`